src/logic/otp2/controller.ts

```diff
@@ -134,10 +137,16 @@ function getTripPatternsVariables(params: Otp2GetTripPatternParams): any {
     }
 }
 
-function getTripPatternsQuery(params: Otp2GetTripPatternParams): GraphqlQuery {
+function getTripPatternsQuery(
+    params: Otp2GetTripPatternParams,
+    comment: string,
+): GraphqlQuery {
     return {
         query: JOURNEY_PLANNER_QUERY,
-        variables: getTripPatternsVariables(params),
+        variables: {
+            ...getTripPatternsVariables(params),
+            comment,
```

Comment:
Denne variabelen blir vel ikkje brukt i queryet eller av JourneyPlanner. Kva er det til? Er det for å gjenkjenne det i debug i browseren? Her ser det fort ut som at det er noko JourneyPlanner krever.

Reply:
Stemmer, og grunnen til at det ligger der er at det da automatisk kom ut i klienten. Jeg hadde det først side om side med variables men da ble det ikke logget. Vi kan evt skrive om klienten også.